frontend/lib/transactions.ts

```diff
@@ -2,6 +2,7 @@ export enum TransactionKeys {
   UNKNOWN = 'unknown',
   ORDER_SUBMISSION = 'orderSubmission',
   ORDER_CANCELLATION = 'orderCancellation',
+  MASS_ORDER_CANCELLATION = 'massOrderCancellation',
```

Comment:
This is currently an accurate representation of possible transaction keys within vega (i.e. the child of the command key in a transaction). I think it's probably better to keep it that way; can we remove this and just put the string into your function above please.

```suggestion
```